migrations/helpers/shared_helper.py

```diff
@@ -1,8 +1,23 @@
+from datetime import datetime
```

Comment:
I would make the name of this file less generic, as it is only relevant to the update_filtered_positives migration, not all migrations. Wouldn't want someone to put some completely unrelated code in in future.

Reply:
Ah I see it's already got some shared stuff in it. In which case I would make a new file to put these in

Reply:
The methods that I extracted are used in the `dart_samples_update_helper` and the `update_filtered_positives_helper`: they are no less shared than the methods that already existed in the file. It was my intention to use this file for the refactoring of the migrations https://github.com/sanger/crawler/issues/173